packages/html-reporter/src/reportView.tsx

```diff
@@ -46,7 +47,7 @@ export const ReportView: React.FC<{
   return <div className='htmlreport vbox px-4 pb-4'>
     <main>
       {report?.json() && <HeaderView stats={report.json().stats} filterText={filterText} setFilterText={setFilterText}></HeaderView>}
-      {report?.json().metadata && <MetadataView {...report?.json().metadata!} />}
+      {report?.json().metadata && <HideIfErrors><MetadataView {...report?.json().metadata!} /></HideIfErrors>}
```

Comment:
What is <HideIfErrors> used for here?

Reply:
https://github.com/microsoft/playwright/pull/13837/files#diff-85123fb439c927ffac44a1e35b5fea6df9e24ecbe03b2575f65f41dea1a752bcR841

Reply:
If someone manually puts weirdness into the metadata field via the config (which they can already use and access today), we don't want to crash the HTML Report.